fortitude/src/rules/mod.rs

```diff
@@ -125,6 +125,7 @@ pub fn code_to_rule(category: Category, code: &str) -> Option<(RuleGroup, Rule)>
         (Style, "201") => (RuleGroup::Stable, Ast, Optional, style::implicit_none::SuperfluousImplicitNone),
         (Style, "211") => (RuleGroup::Preview, Ast, Optional, style::file_contents::MultipleModules),
         (Style, "212") => (RuleGroup::Preview, Ast, Optional, style::file_contents::ProgramWithModule),
+        (Style, "220") => (RuleGroup::Preview, Ast, Optional, style::functions::FunctionMissingResult),
```

Comment:
A super pedantic thing: Could this be rule 221 instead? The reason we never end with a zero is because we don't want to have rule codes like `S000`, and for consistency with each of the other groupings such as `S01x`, `S21x`, etc I've been avoiding ending any rule codes with a 0. The only exception is `E000`, which is only raised if a file can't be accessed for some reason.

I thought this was something I'd picked up from Ruff, but on double checking it seems they're all over the place with rule codes, so I'm now not sure where I got this from.

Reply:
Yea, I just forgot the standard was starting at 1 - changed now.